docs/introduction/faq-and-common-issues.md

```diff
@@ -125,4 +126,21 @@ Otherwise, the logic is in facades (resp. the facades are used as delegates to o
 See [Friendly URL](/docs/introduction/friendly-url.md) article.
 
 ## How can I create Front-end Breadcrumb navigation?
-See [Front-end Breadcrumb Navigation](/docs/introduction/front-end-breadcrumb-navigation.md) article.
\ No newline at end of file
+See [Front-end Breadcrumb Navigation](/docs/introduction/front-end-breadcrumb-navigation.md) article.
+
+## Do you have any tips how to debug emails during development in Docker?
+Yes we have, you can easily use [`djfarrelly/MailDev`](https://github.com/djfarrelly/MailDev) library that provides you web UI where you can see the emails including their headers:
+1. In your `docker-compose.yml`, change the `smtp-server` service:
+    ```diff
+     smtp-server:
+   -        image: namshi/smtp:latest
+   +        image: djfarrelly/maildev
+             container_name: shopsys-framework-smtp-server
+   +        ports:
+   +            - "8025:80"
+    ```
+1. Run `docker-compose up -d`
+1. Now you are able to see all the application emails in the inbox on `http://127.0.0.1:8025`
```

Comment:
one last thing :D please make `http://127.0.0.1:8025` a link

Reply:
fixed in https://github.com/shopsys/shopsys/pull/1252/commits/02c16b9f072be168eee4b7a712fdaa7d3f7cfd53